cluster/ci/config/prow/config.yaml

```diff
@@ -360,6 +362,16 @@ presubmits:
     rerun_command: "/test service-catalog"
     trigger: "((?m)^/test service-catalog,?(\\s+|$))"
 
+  - name: test_pull_request_openshift_service_catalog
```

Comment:
If this job is meant to run on PRs to openshift/service-catalog, you need to update this to:
```yaml
openshift/service-catalog:
- name: test_pull_request_openshift_service_catalog
<- snip ->
```

Reply:
Hrm, also I am not sure what happens when we try to trigger the same job from two different repos.

Reply:
These are different jobs though, I'm confused by your last comment. I think for now the idea is to have openshift commits run the catalog job, but you've made a good point of considering to run CI on openshift/service-catalog commits as well. Will circle back soon.